File: Java/bee 1134.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int qtdAlcool = 0;
        int qtdGasolina = 0;
        int qtdDiesel = 0;
        int input = 0;
        while(input != 4){
            input = sc.nextInt();
            switch (input){
                case 1:
                    qtdAlcool++;
                    break;
                case 2:
                    qtdGasolina++;
                    break;
                case 3:
                    qtdDiesel++;
            }
        }

        System.out.printf("MUITO OBRIGADO\nAlcool: %d\nGasolina: %d\nDiesel: %d\n", qtdAlcool, qtdGasolina, qtdDiesel);
    }
}

File: C++/bee 1134.cpp
#include <iostream>
#include <iomanip>
#include <stdio.h>

using namespace std;

int main()
{
    int qtdAlcool = 0;
    int qtdGasolina = 0;
    int qtdDiesel = 0;
    int n = 0;
    while(n != 4){
        cin >> n;
        switch(n){
        case 1:
            qtdAlcool++;
            break;
        case 2:
            qtdGasolina++;
            break;
        case 3:
            qtdDiesel++;
            break;
        }
    }
    cout << "MUITO OBRIGADO" << endl;
    cout << "Alcool: " << qtdAlcool << endl << "Gasolina: " << qtdGasolina << endl << "Diesel: " << qtdDiesel << endl;
}
